hera_mc/mc_session.py

```diff
@@ -252,6 +252,56 @@ def get_server_status(self, subsystem, starttime, stoptime=None, hostname=None):
 
         return status_list
 
+    def add_subsystem_error(self, time, subsystem, severity, log):
+        """
+        Add a new subsystem subsystem_error to the M&C database.
+
+        Parameters:
+        ------------
+        subsystem: string
+            name of subsystem.
+        time: astropy time object
+            time of this error report
+        log: string
```

Comment:
Missing docstring for "severity", which is important since we should document that lower numbers are more severe.